css-color/Overview.bs

```diff
@@ -1872,18 +1890,8 @@ Predefined colorspaces: ''srgb'', ''dci-p3'', and ''rec2020''.</h3>
 				<tr><th>Green chromaticity</th><td>x=0.170 y=0.797</td></tr>
 				<tr><th>Blue chromaticity</th><td>x=0.131 y=0.046</td></tr>
 				<tr><th>White chromaticity</th><td>x=0.3127 y=0.3290 (D65)</td></tr>
-				<tr><th>Transfer function</th><td>see below</td></tr>
+				<tr><th>Transfer function</th><td>1/2.4</td></tr>
```

Comment:
@cabanier ... because ITU Rec.2087 provides two data points that can be useful to implementers and authors:

(a) conversion from Rec 2020 to Rec 709
(b) discusses the two distinct cases, each of which use a different EOTF: "where the goal is to preserve colours seen on a Rec. 709 display" and "In the case where the source is a direct camera output and the goal is to match the colours of a direct Rec. 2020 camera output"